examples/nim.cfg

```diff
@@ -1,44 +1,103 @@
---passC:"-I../../wxWidgets-3.0.2/lib/gcc_lib/mswu"
+@if vcc:
+--passC:"/I..\\..\\wxWidgets-3.0.2\\include"
```

Comment:
@Araq I have co-developed with @PMunch on this, and therefore I have created a PR for @PMunch (so that everything end up here in the end anyways).

But if you want you can have a look at https://github.com/PMunch/wxnim/pull/2 to see my proposal for moving the compile options to `wx.nim` (or rather `wxCompile.nim`). I have also introduced a symbol define `wxWidgetsPath` that you can use to specify where the wxWidgets include and lib directories are on your system. That should take care of a lot of the cross-platform interoperability issues.